jira-slack-server-integration/jira-slack-server-integration-plugin/src/main/resources/atlassian-plugin.xml

```diff
@@ -123,6 +124,11 @@
         <resource name="slack-settings.soy" type="soy" location="slack-soy/settings/settings.soy"/>
     </web-resource>
 
+    <web-resource key="slack-backbone-resource">
+        <dependency>jira.webresources:backbone-1.3.3</dependency>
```

Comment:
Will it work on all Jira versions starting with 8.15?

Reply:
Yes, it will work